iothub/device/src/Transport/Mqtt/MqttIotHubAdapter.cs

```diff
@@ -288,11 +288,18 @@ private async Task Connect(IChannelHandlerContext context)
                     Debug.Assert(this.mqttTransportSettings.ClientCertificate != null);
                 }
 
+                string usernameString = $"{this.iotHubHostName}/{id}/?api-version={ClientApiVersionHelper.ApiVersionString}&DeviceClientType={Uri.EscapeDataString(this.productInfo.ToString())}";
+
+                if (!this.mqttTransportSettings.AuthenticationChain.IsNullOrWhiteSpace())
+                {
+                    usernameString += $"&AuthChain={Uri.EscapeDataString(this.mqttTransportSettings.AuthenticationChain)}";
```

Comment:
should this property be auth-chain? It seems both formats are used api-version and AuthChain

Reply:
sure thing